src/libs/localStore/storage.ts

```diff
@@ -24,21 +28,35 @@ export enum StorageKeys {
   GeolocationTrackingConfig = 'CozyGPSMemory.TrackingConfig',
   Activities = 'CozyGPSMemory.Activities',
   FirstTimeserie = 'CozyGPSMemory.FirstTimeserie',
-  ServiceWebhookURL = 'CozyGPSMemory.ServiceWebhookURL'
+  ServiceWebhookURL = 'CozyGPSMemory.ServiceWebhookURL',
+  Oauth = '@cozy_AmiralAppOAuthConfig',
+  Cookie = '@cozy_AmiralAppCookieConfig',
+  SessionCreated = '@cozy_AmiralAppSessionCreated'
+}
+
+/*
+  Linked to device.
+  Not removed at logout.
+*/
+export enum DevicePersistedStorageKeys {
+  OnboardingPartner = '@cozy_AmiralAppOnboardingPartnerConfig',
+  ClouderyEnv = '@cozy_AmiralAppClouderyEnvConfig',
+  Bundle = '@cozy_AmiralAppBundleConfig'
 }
 
 export type IconsCache = Record<string, { version: string; xml: string }>
 
 export interface StorageItems {
   biometryActivated: boolean
   biometryType?: BiometryType
-  sessionCreatedFlag: string
   iconCache: IconsCache
   firstTimeserie: FirstTimeserie
+  onboardingPartner: object
+  clouderyEnv: string
 }
 
 export const storeData = async (
-  name: StorageKeys,
+  name: StorageKeys | DevicePersistedStorageKeys,
```

Comment:
nit: here I would rename `StorageKeys` as `CozyPersistedStorageKeys` and I would create a new `StorageKeys` that is the union of both `CozyPersistedStorageKeys` and `DevicePersistedStorageKeys`

Reply:
Would you also rename `StorageKeys` as `CozyPersistedStorageKeys` in every file ? So that we can quickly distinguish in our code if it is cozy persisted or device persisted ? Or just here ? 

Asking because it will modify 200+ files.

Reply:
Did it (with `rename symbol` 👍)